plugins/item_tasks/web_client/views/SelectTaskView.js

```diff
@@ -0,0 +1,66 @@
+import SearchFieldWidget from 'girder/views/widgets/SearchFieldWidget';
+import View from 'girder/views/View';
+
+import 'girder/utilities/jquery/girderEnable';
+import 'girder/utilities/jquery/girderModal';
+
+import SelectTaskViewDialogTemplate from '../templates/selectTaskViewDialog.pug';
+import selectTaskViewDescriptionTemplate from '../templates/selectTaskViewTargetDescription.pug';
+
+import '../stylesheets/selectTaskView.styl';
+
+/**
+ * A dialog for creating tasks from a specific item.
+ */
+var SelectTaskView = View.extend({
+    events: {
+        'submit #g-select-task-form': function (e) {
+            e.preventDefault();
+            this.$('.g-submit-select-task').girderEnable(false);
+
+            this.trigger('g:selected', {
+                itemId: this.item.id,
+                taskId: this.taskId
+            });
+        }
+    },
+
+    initialize: function (settings) {
+        this.item = settings.item;
+
+        this.searchWidget = new SearchFieldWidget({
+            placeholder: 'Start typing a task name...',
+            types: ['item'],
```

Comment:
This should definitely be a more specific search (confined to just item_tasks), probably utilizing whatever gets added by https://github.com/girder/girder/pull/2004

Alternatively (or perhaps additionally), it may be better to display a selectable list of available item_tasks. This should utilize the filtering added by https://github.com/girder/girder/pull/2133 .